<launch>
 	<arg name="veh"/>
 	<arg name="config" default="baseline"/>
 	<arg name="pkg_name" value="apriltags"/>
 	<arg name="node_name" value="apriltags_preprocessing_node"/>
 	<arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman"/>
 	
 	<group ns="$(arg veh)">
         <node pkg="$(arg pkg_name)" type="$(arg node_name).py" name="$(arg node_name)" output="screen" clear_params="true" required="true">
         <rosparam command="load" file="$(find duckietown)/config/$(arg config)/$(arg pkg_name)/$(arg node_name)/$(arg param_file_name).yaml"/>
         </node>
 	</group>
 </launch>
